client/gutenberg/extensions/simple-payments/edit.js

```diff
@@ -325,7 +325,7 @@ class SimplePaymentsEdit extends Component {
 			return (
 				<div className="simple-payments__loading">
 					<ProductPlaceholder
-						ariaBusy="true"
+						aria-busy="true"
```

Comment:
`ariaBusy` maps to `ariabusy` attribute which is incorrect. `aria-busy` should produce the correct attribute `aria-busy`.

https://reactjs.org/docs/accessibility.html#wai-aria